resources/dicts/events/lifegen_events/murder.json

```diff
@@ -362,12 +367,12 @@
         "You give an indignant huff as v_c once again tries to 'help' out with your duties. Who is the deputy here, you or v_c? Just because they are an elder does not mean they are suddenly imbued with the knowledge of StarClan! Fed up, you ask v_c to take a peaceful walk outside the camp. Hours later, when asked about the whereabouts of v_c, you simply shake your head and murmur that you have no idea."
     ],
     "murder leader kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, having already gone to StarClan.",
         "Like a snake with a charming voice, you guide v_c into the corners of the Clan's borders, giving them a false promise of 'teaching' them a new move only they can witness- what's cooler than training with the leader of the Clan! v_c promises to take it to their grave. Unfortunately, you made sure they followed your exact orders.",
-        "You offer to take v_c onto a grand and exciting adventure out of the nursery. They agree, exhilarated at the thought of accompanying the leader on a journey. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind as their startled cries call out your name from behind. You loop back to camp, alone, and no one asks where you've been.",
+        "You offer to take v_c onto a grand and exciting adventure out of the nursery. They agree, exhilarated at the thought of accompanying the leader on a journey. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been.",
```

Comment:
this can be changed to match the previous one

"You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out, and the two of you run freely through the territory. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been."